code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -338,6 +338,10 @@ Credit where due:
 	CLOCKCULTCHANGELOG\
 	</ul>\
 	<hr>\
+	<li><b>Brass Flasks:</b> These contain 30u of Zelus oil, this oil can be used to help stun people, and more son on Blood Cult\
+	It can also be used to splash onto metal to make brass. Grinding bass will give you some oil back as well.\
+	You can make more brass flasks to be used as throwing weaponds if you so wish, these will not have any oil in them. They have one throw before braking making them rather useless.\
+	Lastly the oil can be drank for a bost in combat, gives you stamina as well as heals quickly!\
```

Comment:
"bost" -> "boost"

I'd also add the word "it" after the comma, but that's more grammar than anything